docs/monthday.md

```diff
@@ -13,20 +13,22 @@ A `Temporal.MonthDay` can be converted into a `Temporal.Date` by combining it wi
 
 ## Constructor
 
-### **new Temporal.MonthDay**(_isoMonth_: number, _isoDay_: number) : Temporal.MonthDay
+### **new Temporal.MonthDay**(_isoMonth_: number, _isoDay_: number, _refISOYear_: number = 1972) : Temporal.MonthDay
```

Comment:
this is a total nit, but it'd be *real nice* if instead of hardcoding 1972 here, this was something like `getNextLeapYear(new Date(0).getFullYear)`, assuming `getNextLeapYear` was a thing that exists.

Reply:
How would you feel about removing the hardcoded "1972" from this documentation page and the spec, instead describing how the value should be derived, but leaving it hardcoded in the polyfill? I mean we could use something like `Math.ceil(new Date(0).getUTCFullYear() / 4) * 4` but that's not correct for multiples of 100 and it's not like `new Date(0)` is ever going to be a different value...

Reply:
the polyfill doesn't matter, so sure, that's fine.